crates/pow/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+authors.workspace      = true
+description            = "Miden faucet's PoW"
+edition.workspace      = true
+homepage.workspace     = true
+keywords               = ["faucet", "miden", "pow"]
+license.workspace      = true
+name                   = "miden-faucet-pow"
```

Comment:
I would probably make this crate a bit more generic and we could name it `miden-pow-rate-limiter` or something like that.